app/services/dependency-resolver.js

```diff
@@ -1,6 +1,6 @@
 import Ember from 'ember';
 
-const EMBER_VERSIONS = ['2.4.3', '2.3.2', '2.2.2', '2.1.2', '2.0.3', '1.13.13', '1.12.2'];
+const EMBER_VERSIONS = ['2.4.4', '2.3.2', '2.2.2', '2.1.2', '2.0.3', '1.13.13', '1.12.2'];
```

Comment:
Unrelated to this PR, it might be nice to not have to hard code this list.  Maybe we can come up with a way detect the list of valid versions (i.e. a `.json` file we can grab from builds.emberjs.com or something)...